docs/domains/Domain.md

```diff
@@ -193,6 +196,21 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | `annotations` | Map | The annotations to be attached to generated resources. |
 | `labels` | Map | The labels to be attached to generated resources. The label names must not start with 'weblogic.'. |
 
+### Model
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `configMapName` | string | WDT config map name |
+| `domainType` | string | WDT domain type: Legal values: WLS, RestrictedJRF, JRF |
+| `encryptionSecret` | [Secret Reference](k8s1.13.5.md#secret-reference) | WDT encryption key passphrase secret. |
+
+### Opss
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `keyWalletConfigMap` | string | The name of the config map containing the OPSS key wallet file |
```

Comment:
Now a secret and renamed to   walletFileSecret

Reply:
I think we agreed to change this to walletFileSecret and walletPasswordSecret per slack thread https://proddev-paas-fmw.slack.com/archives/CKFA73SNM/p1580504991015300